src/Imath/ImathVec.h

```diff
@@ -72,15 +72,24 @@ template <class T> class IMATH_EXPORT_TEMPLATE_TYPE Vec2
     IMATH_HOSTDEVICE constexpr Vec2 (T a, T b) IMATH_NOEXCEPT;
 
     /// Copy constructor
+#if IMATH_VEC_BE_TRIVIALLY_RELOCATABLE != 0
+    IMATH_HOSTDEVICE constexpr Vec2 (const Vec2& v) IMATH_NOEXCEPT = default;
```

Comment:
```suggestion
    IMATH_HOSTDEVICE Vec2 (const Vec2&) = default;
```
Defaulted special member functions are "constexpr auto" and "noexcept auto" by default; you don't have to (and therefore _should not_) write those modifiers in the code. Let the compiler figure it out.